ui/tests/integration/components/clients/config-test.js

```diff
@@ -18,15 +18,15 @@ module('Integration | Component | client count config', function (hooks) {
     this.router = this.owner.lookup('service:router');
     this.transitionStub = sinon.stub(this.router, 'transitionTo');
     const store = this.owner.lookup('service:store');
-    this.createModel = (enabled = 'enable', reporting_enabled = false, minimum_retention_months = 24) => {
+    this.createModel = (enabled = 'enable', reporting_enabled = false, minimum_retention_months = 48) => {
```

Comment:
this value is 48 now so I thought updating the default for the test scenarios made sense